cloudmarker/alerts/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""A package for alert plugins packaged with this project.
+
+This package contains alert plugins that are packaged as part of this
+project. The alert plugins implement a function named ``write()`` that
+accepts input records and typically sends them to an alerting
+destination. The alert plugins also implement and a function named
```

Comment:
...also implement ~~and~~ a function...

Reply:
Thanks for noticing this.

[Fixed](https://github.com/cloudmarker/cloudmarker/pull/85/files#diff-1c9c69bbb7b60c962639f92a74fe27aaR6).